Games/types/Mafia/const/Priority.js

```diff
@@ -39,9 +41,13 @@ const PRIORITY_GUN_GIVER = PRIORITY_ITEM_GIVER_DEFAULT
 const PRIORITY_ORANGE_GIVER = PRIORITY_ITEM_GIVER_DEFAULT
 const PRIORITY_PROBE = PRIORITY_ITEM_GIVER_DEFAULT
 
+const PRIORITY_MESSAGE_GIVER_DEFAULT = -50
+const PRIORITY_CONFIRM_SELF = PRIORITY_MESSAGE_GIVER_DEFAULT
+
 const PRIORITY_INVESTIGATIVE_DEFAULT = -10
 const PRIORITY_ALIGNMENT_LEARNER = PRIORITY_INVESTIGATIVE_DEFAULT
 const PRIORITY_ROLE_LEARNER = PRIORITY_INVESTIGATIVE_DEFAULT
+const PRIORITY_LEARN_VISITORS = PRIORITY_INVESTIGATIVE_DEFAULT
```

Comment:
I think learn visitors has to be the last action after tracker and watcher, or gramps cannot see tracker and watcher move.